platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/merkle/VirtualRootNode.java

```diff
@@ -1283,7 +1286,13 @@ public void deserialize(final SerializableDataInputStream in, final Path inputDi
             keySerializer = in.readSerializable();
             valueSerializer = in.readSerializable();
         }
-        cache = in.readSerializable();
+        if (version < ClassVersion.VERSION_3_NO_NODE_CACHE) {
+            // Future work: once all states are version 3 or later, this code branch can be
```

Comment:
Is there a ticket already?

Reply:
There is no ticket yet, but I am already taking care of these lines in one of my future PRs (#16488), so it will not be lost.